exporters/zipkin/model.go

```diff
@@ -51,7 +51,9 @@ func init() {
 	}
 }
 
-func toZipkinSpanModels(batch []tracesdk.ReadOnlySpan) []zkmodel.SpanModel {
+// ToZipkinSpanModels converts OpenTelemetry spans into Zipkin model spans.
+// This is used for exporting to Zipkin compatible tracing services.
+func ToZipkinSpanModels(batch []tracesdk.ReadOnlySpan) []zkmodel.SpanModel {
```

Comment:
If we plan to export this I would like to put a bit more thought into the name. I think we can use the package name as part of the function name and just rename this as follows.

```suggestion
func SpanModels(batch []tracesdk.ReadOnlySpan) []zkmodel.SpanModel {
```

thoughts?

Reply:
I like that name better also, I have updated the PR.